### DISCUSSION TITLE
How to make a HTTP request to an external API service from the connector ?

### DISCUSSION THREAD

**Discussion by imonbayazid** [International-Data-Spaces-Association/DataspaceConnector#76] [Q&A] [answered]
Hello,
I need to make an HTTP call to get some values using GET/POST call to an external service that I have developed.
Is it possible from the connector and if so then how can I achieve that?

Thanks

**Comment by HeinrichPet** [comment, marked as answer]
As discussed in our meeting today, the connector will not actively push metadata to an service.
But you can request all offers via GET /api/offers, getting the metadata you need and add additional metadata via PUT /api/offers/{id}.
